Week10/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/di/module/AppModule.kt

```diff
@@ -0,0 +1,55 @@
+package com.e.blockbusterrecycler.di.module
```

Comment:
Really nice work on setting up the modules below, I think you did a nice job identifying dependencies, declaring them in the modules, and separating the modules into different files as instructed.